enterprise/packages/billing/package.json

```diff
@@ -47,8 +48,5 @@
     "@nestjs/swagger": "7.4.0",
     "@nestjs/throttler": "6.2.1",
     "@novu/dal": "workspace:*"
-  },
-  "optionalDependencies": {
-    "@novu/ee-auth": "workspace:*"
```

Comment:
Billing is always used as an enterprise package therefore it always need `@novu/ee-auth`